### ISSUE TITLE
When updating value via bound value, no revalidation occurs

### ISSUE THREAD

**Issue by rburgst** [ciscoheat/sveltekit-superforms#131]
**Description**
When extracting a form component and updating a bound value (which is bound to `$form.name`) then no automatic revalidation occurs.

To reproduce see 
https://stackblitz.com/edit/sveltekit-superforms-bug-reporting?file=src%2Froutes%2F%2Bpage.server.ts,src%2Froutes%2F%2Bpage.svelte](https://stackblitz.com/edit/sveltekit-superforms-testing-hxuyyv?file=src/input-field.svelte)

1. Enter a name which a lowercase first letter, e.g. `foo` => you get a validation warning
2. Click the "reset" button (this will insert a valid name). 

**Expected:** an automatic re-validation should happen when the store value for `$form.name` changes
**Actual:** nothing happens, the previous validation error is still shown even though the state in the `$form.name` is already updated.

**Comment by rburgst** [comment]
This is the component source
```svelte
<script lang="ts">
  export let field
  export let value
  export let errors
  export let label

</script>

<label>
  {label}
  <input
    type="text"
    data-invalid={errors}
    bind:value={value}
  />
</label>
{#if errors}<span class="invalid">{errors}</span>{/if}

<button on:click={(e) => {
  e.preventDefault();
  value = "I am valid";
}}>reset</button>
<style lang="scss">
  .invalid {
    color: orangered;
  }
</style>
```

As you can see then click on the `reset` button immediately updates the bound form field.

**Comment by ciscoheat** [comment]
Tracked in #114, will be fixed in the next patch version.

**Comment by ciscoheat** [comment]
Should be fixed now with v0.8.5, can you confirm?

**Comment by rburgst** [comment]
tested with 0.8.6 and it seems to work fine. Thanks for the rapid support!
